config_test.go

```diff
@@ -9,6 +9,19 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 )
 
+func (r *Repo) Equal(x interface{}) bool {
```

Comment:
This Equal function introduction seems unrelated to perrepo purge option. It makes sense to move it to a separate commit (but keep it in this PR).